lambda-assets/extensions/cf-convert-query-string/index.ts

```diff
@@ -0,0 +1,39 @@
+import { URLSearchParams } from 'url'
+export async function lambdaHandler(event: any) {
+    /**
+     * A magic line that can achieve the effect of injecting environment variables.
+     * Go check the definition of the L@E.
+     * */
```

Comment:
```suggestion
    /**
     * Whitelisted request arguments that will be converted into additional headers
     * `NEEDED_KEYS` is the placeholder for `args` that will be bundled and injected by esbuild.
     * */
```

Actually they are not `environment variables`, let's change the description as suggested.